lms/djangoapps/ccx/views.py

```diff
@@ -349,6 +349,9 @@ def visit(node, depth=1):
         widgets, which use text inputs.
         """
         for child in node.get_children():
+            # in case the children are visible to staff only, skip them
```

Comment:
Yes, I followed that, but it's important that the code is self documenting. Later contributors need to understand why things are working a certain way.